translation/source/mobile.xml

```diff
@@ -5,4 +5,39 @@
+  <string name="puzzleStreakAbortWarning" comment="Shown as a subtitle in the confirmation dialog for aborting puzzle streak.">You will lose your current streak and your score will be saved.</string>
+  <string name="puzzleStormNothingToShow" comment="Shown when the user has not played any puzzle storm runs yet">Nothing to show. Play some runs of storm</string>
+  <string name="sharePuzzle" comment="This is shown in the puzzle screen. When pressed, the user can share the puzzle.">Share this puzzle</string>
+  <string name="shareGameURL" comment="Menu action that shares the URL of the current game">Share game URL</string>
+  <string name="shareGamePGN" comment="Menu action that shares the PGN of a game">Share PGN</string>
+  <string name="sharePositionAsFEN" comment="Menu action that shares the current position as a FEN string">Share position as FEN</string>
+  <string name="showVariations" comment="Shown in the analysis screen.">Show variations</string>
+  <string name="hideVariation" comment="Shown in the analysis screen.">Hide variation</string>
+  <string name="showComments" comment="Shown in the analysis screen.">Show comments</string>
+  <string name="puzzleStormConfirmEndRun" comment="Text of the confirmation dialog when ending a puzzle storm run">Do you want to end this run?</string>
+  <string name="puzzleStormFilterNothingToShow" comment="Shown when the user has no puzzles matching the filter criteria">Nothing to show, please change the filters</string>
+  <string name="cancelTakebackOffer" comment="Menu action that cancels a previous takeback offer">Cancel takeback offer</string>
+  <string name="cancelDrawOffer" comment="Menu action that cancels a previous draw offer">Cancel draw offer</string>
+  <string name="waitingForOpponentToJoin" comment="This is after the user has created a game and is waiting for an opponent to join.">Waiting for opponent to join...</string>
+  <string name="blindfoldMode" comment="This button toggles blindfold mode">Blindfold</string>
+  <string name="cannotLoadGameAnalysis" comment="This is shown when the app fails to load the game analysis. %s contains a description of the error.">Cannot load game analysis: %s</string>
+  <string name="liveStreamers" comment="This is shown as a heading in the streamer screen">Live streamers</string>
+  <string name="customGameJoinAGame" comment="This is shown in the custom game screen. Will open the game lobby.">Join a game</string>
+  <string name="correspondenceClearSavedMove" comment="This is shown in the correspondence game screen. Will clear the saved move.">Clear saved move</string>
+  <string name="somethingWentWrong" comment="This is a generic text to be shown when something unexpected happens, e.g. failing to load data from the server.">Something went wrong. If the issue persists, consider opening an issue on GitHub or reaching out in the lichess Discord server.</string>
```

Comment:
Actually I'd leave "Something went wrong".  (Sorry, I know I suggested we could improve the message).

This message is intended mainly for network errors, so it's not relevant to talk about GitHub issues here.